staff/lila-petri/spotify-template/desktop/index.html

```diff
@@ -0,0 +1,80 @@
+                    </path>
+                </svg>
+            </div>
+            <div class="navegation">
+                    <div class="navegation__item">
+                        <svg class="img" viewBox="0 0 512 512">
+                            <path d="M448 463.746h-149.333v-149.333h-85.334v149.333h-149.333v-315.428l192-111.746 192 110.984v316.19z"></path>
+                        </svg>
+                        <span class="text text--color-inicio">Inicio</span>
+                    </div>
+                    <div class="navegation__item">
+                        <svg class="img" viewBox="0 0 512 512">
+                            <path d="M349.714 347.937l93.714 109.969-16.254 13.969-93.969-109.969q-48.508 36.825-109.207 36.825-36.826 0-70.476-14.349t-57.905-38.603-38.603-57.905-14.349-70.476 14.349-70.476 38.603-57.905 57.905-38.603 70.476-14.349 70.476 14.349 57.905 38.603 38.603 57.905 14.349 70.476q0 37.841-14.73 71.619t-40.889 58.921zM224 377.397q43.428 0 80.254-21.461t58.286-58.286 21.461-80.254-21.461-80.254-58.286-58.285-80.254-21.46-80.254 21.46-58.285 58.285-21.46 80.254 21.46 80.254 58.285 58.286 80.254 21.461z"></path>
+                        </svg>
+                        <span class="text text--color">Buscar</span>
+                    </div>
+                    <div class="navegation__item">
+                        <svg class="img" viewBox="0 0 512 512" >
+                            <path d="M291.301 81.778l166.349 373.587-19.301 8.635-166.349-373.587zM64 463.746v-384h21.334v384h-21.334zM192 463.746v-384h21.334v384h-21.334z"></path>
+                        </svg>
+                        <span class="text text--color">Tu biblioteca</span>
+                    </div>
+            </div>
+            <div class="navegation">
+                <div class="navegation__item">
+                    <h2 class="text text--color">ESCUCHADOS <br> RECIENTEMENTE</h2>
+                </div>
+                <div class="navegation__item">
+                    <h4 class="text text--color">Temazos Chill<br>PLAYLIST</h4>
+                </div>
+                <div class="navegation__item">
+                    <h4 class="text text--color">Eco<br>ÁLBUM</h4>
+                </div>
+                <div class="navegation__item">
+                    <h4 class="text text--color">Despierta y Sonrie<br>ÁLBUM</h4>
+                </div>
+            </div>
+        </div>
+        <div class="navbar__footer">
+            <div class="navegation">     
```

Comment:
@mlila with BEM, maybe you should rename them as it follows:
`footer`, `footer__navbar`, `footer__navbar-item`, `footer__navbar-item-text`, `footer__navbar-item-avatar`